.github/workflows/cross-compiles.yml

```diff
@@ -210,3 +210,9 @@ jobs:
         make test HARNESS_JOBS=${HARNESS_JOBS:-4} \
                   TESTS="${{ matrix.platform.tests }} -test_afalg" \
                   QEMU_LD_PREFIX=/usr/${{ matrix.platform.arch }}
+    - name: make evp tests
+      if: github.event_name == 'pull_request' && matrix.platform.tests != 'none'
+      run: |
+        make test HARNESS_JOBS=${HARNESS_JOBS:-4} \
+                  TESTS="test_evp*" \
```

Comment:
This at least does something.. It probably wouldnt have found the issue with my first 'fix'. but better than not doing anything.

Reply:
It would find it. The CI on this PR fails without the fix.